internal/app/machined/internal/sequencer/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -152,17 +175,7 @@ func (d *Sequencer) Boot() error {
 
 // Shutdown implements the Sequencer interface.
 func (d *Sequencer) Shutdown() error {
-	content, err := config.FromFile(constants.ConfigPath)
-	if err != nil {
-		return err
-	}
-
-	config, err := config.New(content)
-	if err != nil {
-		return err
-	}
-
-	phaserunner, err := phase.NewRunner(config, runtime.Shutdown)
+	phaserunner, err := phase.NewRunner(nil, runtime.Shutdown)
```

Comment:
There was no need for shutdown to read the config file from what I could tell.

Reply:
what if we need to access it in the shutdown sequence some time in the future? if we get to shutdown phase, we should have got through the boot phase successfully, so config should be in `/boot`?

Reply:
fixed this in amended commit